opcua/adapter.go

```diff
@@ -33,24 +34,24 @@ type Service interface {
 	RemoveChannel(ctx context.Context, chanID string) error
 
 	// ConnectThing creates thingID:channelID route-map
-	ConnectThing(ctx context.Context, chanID, thingID string) error
+	ConnectThing(ctx context.Context, chanID string, thingIDs []string) error
 
 	// DisconnectThing removes thingID:channelID route-map
-	DisconnectThing(ctx context.Context, chanID, thingID string) error
+	DisconnectThing(ctx context.Context, chanID string, thingIDs []string) error
 
 	// Browse browses available nodes for a given OPC-UA Server URI and NodeID
-	Browse(ctx context.Context, serverURI, namespace, identifier string) ([]BrowsedNode, error)
+	Browse(ctx context.Context, serverURI, namespace, identifier, identifierType string) ([]BrowsedNode, error)
 }
 
 // Config OPC-UA Server.
 type Config struct {
 	ServerURI string
 	NodeID    string
-	Interval  string `env:"MG_OPCUA_ADAPTER_INTERVAL_MS"   envDefault:"1000"`
-	Policy    string `env:"MG_OPCUA_ADAPTER_POLICY"        envDefault:""`
-	Mode      string `env:"MG_OPCUA_ADAPTER_MODE"          envDefault:""`
-	CertFile  string `env:"MG_OPCUA_ADAPTER_CERT_FILE"     envDefault:""`
-	KeyFile   string `env:"MG_OPCUA_ADAPTER_KEY_FILE"      envDefault:""`
+	Interval  string `env:"MG_OPCUA_ADAPTER_INTERVAL_MS"     envDefault:"1000"`
```

Comment:
Remove extra spaces in between
```suggestion
	Interval  string `env:"MG_OPCUA_ADAPTER_INTERVAL_MS" envDefault:"1000"`
```